kubernetes/linux/main.sh

```diff
@@ -1022,21 +1042,14 @@ else
       # add -T 0xFFFF for full traces
       export MDSD_ROLE_PREFIX=/var/run/mdsd-ci/default
       echo "export MDSD_ROLE_PREFIX=$MDSD_ROLE_PREFIX" >> ~/.bashrc
-      if [ -n "$SYSLOG_HOST_PORT" ] && [ "$SYSLOG_HOST_PORT" != "28330" ]; then
-            echo "Updating rsyslog config file with non default SYSLOG_HOST_PORT value ${SYSLOG_HOST_PORT}"
-            if sed -i "s/Port=\"[0-9]*\"/Port=\"$SYSLOG_HOST_PORT\"/g" /etc/opt/microsoft/docker-cimprov/70-rsyslog-forward-mdsd-ci.conf; then
-                  echo "Successfully updated the rsylog config file."
-            else
-                  echo "Failed to update the rsyslog config file."
-            fi
-      else
-            echo "SYSLOG_HOST_PORT is ${SYSLOG_HOST_PORT}. No changes made."
+      if [[ "${GENEVA_LOGS_INTEGRATION}" != "true" ]]; then
+            export MDSD_DEFAULT_TCP_SYSLOG_PORT=28330
```

Comment:
instead of 28330, this should be "$SYSLOG_HOST_PORT" so that we can handle if the different port configured through API.

Reply:
I thought about this, but I wanted to keep things same as what we have right now for 3P scenario. Will make that change seperately. wdyt

Reply:
I don't see any harm doing as part of this PR but its OK if you want to handle separately.